components/accelerator/nvidia/sxid/component.go

```diff
@@ -217,7 +219,7 @@ func (c *SXIDComponent) SetHealthy() error {
 
 func (c *SXIDComponent) updateCurrentState() error {
 	var rebootErr string
-	rebootEvents, err := getRebootEvents(c.rootCtx)
+	rebootEvents, err := os.GetRebootEvents(c.rootCtx, c.eventStore, time.Now().Add(-DefaultRetentionPeriod))
```

Comment:
it would be great if we can move the `GetRebootEvents` out of os component, refer to #510

Reply:
how about `pkg/reboot` and implement `record/read Reboot(ctx, eventStore)`

Reply:
yea, sgtm!

Reply:
@eahydra Done. Will make corresponding changes for control plane once this PR gets merged in.